client/src/components/AddProductModal.jsx

```diff
@@ -0,0 +1,151 @@
+import {
+  Button,
+  Grid,
+  makeStyles,
+  Modal,
+  Paper,
+  Slide,
+  TextField,
+  Typography,
+} from "@material-ui/core";
```

Comment:
Just a preference thing, but you should separate `makeStyles` from the rest of these imports since it's not a component. You can either put it at the top, at the end, or in a separate line